include/d/actor/d_a_tpota.h

```diff
@@ -2,22 +2,36 @@
 #define D_A_TPOTA_H
 
 #include "f_op/f_op_actor.h"
+#include "d/d_particle.h"
 
+class JPABaseEmitter;
 class JPABaseParticle;
 
 class daTpota_c : public fopAc_ac_c {
+
+    struct unknown_struct {
+        JPABaseParticle* ptcl;
+        f32 pos_y;
+    };
+
 public:
     cPhs_State _create();
     bool _delete();
     void make_ripple(cXyz);
-    void check_water_h(JPABaseParticle*, float);
+    int check_water_h(JPABaseParticle*, float);
     void clear_splash();
     void renew_splash();
     bool _execute();
     bool _draw();
 
 public:
-    /* Place member variables here */
-};
+    /* 0x290 */ u8 field_0x290[0x298 - 0x290];
+    /* 0x298 */ JPABaseEmitter* emitters[2];
+    /* 0x2A0 */ cXyz position[2];
+    /* 0x2B8 */ csXyz angle[2];
```

Comment:
```suggestion
    /* 0x298 */ JPABaseEmitter* mpEmitters[2];
    /* 0x2A0 */ cXyz mPositions[2];
    /* 0x2B8 */ csXyz mAngles[2];
```
Member vars start with `m` https://github.com/zeldaret/tww/blob/main/docs/coding_guidelines.md#naming-style